lib/roo_on_rails/routemaster/publisher.rb

```diff
@@ -0,0 +1,57 @@
+require 'roo_on_rails/config'
+require 'routemaster/client'
+
+module RooOnRails
+  module Routemaster
+    class Publisher
+      attr_reader :model, :event
+
+      def initialize(model, event, client: ::Routemaster::Client)
+        @model = model
+        @event = event
+        @client = client
+      end
+
+      def publish?
```

Comment:
The idea is that it's `true` by default and you can then overwrite this in your implementation. You can see this in the example I provide.